app/controllers/support/responsible_bodies_controller.rb

```diff
@@ -17,6 +17,12 @@ def show
     @users = policy_scope(@responsible_body.users).not_deleted.order('last_signed_in_at desc nulls last, updated_at desc')
     @schools = @responsible_body
       .schools
+      .gias_status_open
+      .includes(:device_allocations, :preorder_information)
+      .order(name: :asc)
```

Comment:
If you include these extra lines in the query, it will allow you to refer to `school.user_count` without n+1 queries:
```suggestion
      .gias_status_open
      .includes(:device_allocations, :preorder_information)
      .left_outer_joins(:user_schools)
      .select('schools.*, count(user_schools.id) AS user_count')
      .group('schools.id')
      .order(name: :asc)
```

Reply:
Great spot, not needed in this query but it is for closed schools, thanks.